Sources/SwiftDocC/Infrastructure/Topic Graph/AutomaticCuration.swift

```diff
@@ -177,6 +178,9 @@ public struct AutomaticCuration {
                         variantsTraits.contains(where: { $0.interfaceLanguage == language.id})
                     })
                 }
+                // Don't create too long See Also sections
+                .prefix(15)
```

Comment:
Should we extract this number somewhere we can make it configurable in a follow up?

Reply:
Added in https://github.com/swiftlang/swift-docc/pull/1043/commits/7f5d613e1410e99317bf4641a0466586be7baff6